src/ResourceOperations/ResourceHolder.cs

```diff
@@ -351,7 +383,7 @@ public void Save()
         /// </summary>
         private static bool IsLocalizableString(string key, ResXDataNode dataNode)
         {
-            if (key.StartsWith(">>") || (key.StartsWith("$") && key != "$this.Text"))
+            if ((key.StartsWith(">>") && !key.ToLower().Contains(".caption")) || ((key.StartsWith("$") && key != "$this.Text")))

```

Comment:
Where does the `.caption` come from and what does it mean?

Reply:
it's the same as .name. DevExpress adds [control].caption to give posibilty to add different languajes but they add this resx string with '>>' prefix so, its important show this field when has .caption.